code/commands_generator/eQTL_analysis_commands.ipynb

```diff
@@ -106,9 +106,8 @@
     "kernel": "SoS"
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #6.        --Association_option "TensorQTL" --cwd output2 --run  --yml "" --N 3   
</pre><p>what does it mean by <code>--yml  ""</code> ? if  the default is empty string then it should not need to be specified ... </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/cumc/xqtl-pipeline/pull/442/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='10'/>